ports/samd/boards/SAMD21_XPLAINED_PRO/deploy_xplained_pro.md

```diff
@@ -0,0 +1,25 @@
+The SAMD21 Xplained Pro board is shipped without a bootloader. For use
+with MicroPyhton a suitable bootloader has be be installed first. The
+following procedure has been found to work and be simple:
+
+1. Get the bootloader file bootloader-xplained-pro-v3.16.0-15-gaa52b22.hex
+from https://micropython.org/resources/firmware.
```

Comment:
I think it would be better if this URL was the full URL of the .hex file, like:
```
Get the bootloader from https://micropython.org/resources/firmware/bootloader-xplained-pro-v3.16.0-15-gaa52b22.hex
```